packages/virtual/README.md

```diff
@@ -73,9 +73,9 @@ export default {
   plugins: [
     virtual({
       entry: `
-import batman from 'batcave';
-console.log(batman);
-`
+        import batman from 'batcave';
```

Comment:
because we're using backticks this will add quite a bit more padding to the source of the virtual module. with this being in the README I'd like to avoid that. please do revert this change.

Reply:
Just thought the example in the `README` looks a bit less aligned compared to the rest. Spacing was added for better readability compared to [the current version](https://github.com/rollup/plugins/blob/master/packages/virtual/README.md#using-the-plugin-for-bundle-input):

![Screenshot 2020-04-26 at 01 39 02](https://user-images.githubusercontent.com/1060490/80293347-ee206f00-875e-11ea-9734-acd58ed065b0.png)

But sure it shouldn't have a negative impact so if thats not preferred I can revert it.

Reply:
yes please do revert it

Reply:
Okay changed it back. Hope the other modification of using the `input` field in the first example rather than `entry` was okay 😅